pxl_scripts/px/services_to_ips/services_to_ips.pxl

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Pixie Labs, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+''' Services to and from a list of remote IPs
+
+Finds services that have traffic to and from a list of IP addresses.
+Calculates bandwidth in bytes per second (BPS).
+
+'''
+
+import px
+
+df = px.DataFrame('conn_stats')
+
+# Add IPs of the remote endpoint to filter.
+expr = df.remote_addr == '10.4.1.1'
+expr = expr or df.remote_addr == '10.4.1.2'
```

Comment:
Where are these IPs from?

Reply:
These are place holders.

The values are from my test cluster.

I've revised the format to be consistent with the purpose of asking users to adapt the expression.
Please take a look at the newest commit.